content/en/blog/2022/how-to-get-started-with-opentelemetry-go.md

```diff
@@ -0,0 +1,385 @@
+---
+title: How to Get Started with OpenTelemetry Go
+linkTitle: Get Started with OpenTelemetry & Go
+date: 2022-06-27
+author: Team Aspecto
+canonical_url: https://www.aspecto.io/blog/opentelemetry-go-getting-started/
+---
+
+# How to Get Started with OpenTelemetry Go
+
+In this guide, you will learn hands-on how to create and visualize traces with OpenTelemetry Go without prior knowledge.
+
+We will start with creating a simple to-do app that uses Mongo and the Gin framework. Then, we will send tracing data to Jaeger Tracing and to Aspecto for visualization. You can find all the relevant files in this [Github repository](https://github.com/aspecto-io/opentelemetry-examples/tree/master/go).
+
+!["OpenTelemetry Go The Mandalorian"](https://www.aspecto.io/wp-content/uploads/2022/06/OpenTelemetry-Go-The-Mandalorian-2048x1406.png)
+
+## What to Expect
+
+- Intro to OpenTelemetry
+- Hello world: OpenTelemetry GO example
+  - Create main.go file with Gin and Mongo
+  - Install OpenTelemetry GO
+  - Gin instrumentation: gin.Context
+- Visualization with Jaeger
+  - OpenTelemetry Go and Jaeger Tracing
+
+## Intro to OpenTelemetry
+
+OpenTelemetry is a collection of APIs and SDKs that allows us to collect, export, and generate **traces, logs, and metrics** (also known as the three pillars of observability).
+
+It is a CNCF community-driven open-source project (Cloud Native Computing Foundation, the folks in charge of Kubernetes).
+
+In a cloud-native environment, we use OpenTelemetry (OTel for short) to gather data from our system operations and events. In other words, to instrument our distributed services. This data enables us to understand and investigate our software’s behavior and troubleshoot performance issues and errors.
+
+OpenTelemetry serves as a standard observability framework that captures all data under a single specification. It provides several components, including:
+
+1. APIs and SDKs per programming language for generating telemetry
+2. The OpenTelemetry Collector; receives, processes, and exports telemetry data to different destinations.
+3. [OTLP](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md) protocol for shipping telemetry data
+   To get a deeper understanding of this technology, including its structure and the primary motivation, visit this guide.
+
+For this OpenTelemetry Golang guide, here are the terms you need to know:
+
+- **Span:** The most basic unit. A span represents an event in our system (e.g., an HTTP request or a database operation that spans over time). A span would usually be the parent of another span, its child, or both.
```

Comment:
I'd recommend sticking with the definitions we have in our glossary on the website here. It's fine to copy/paste those definitions, but what you have here does differ a bit from the glossary, which is considered canonical.

FWIW if you see something in the glossary and thing, "hmmm, this could improve" ... please send a PR! I'm more than happy to improve the glossary and incorporate some of what you have here if you think it will be helpful.

Reply:
Hi, Sorry for the late reply. I will check all of your feedback ASAP